satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1300,7 +1409,9 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: reflectance_max
-    long_name: maximum_reflectance_in_segment
+    long_name: TOA Bidirectional Reflectance segment max
+    standard_name: toa_bidirectional_reflectance
```

Comment:
also add `units: %`, set to none in netcdf file.

Reply:
done